<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormGaji</class>
 <widget class="QWidget" name="FormGaji">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="kd_gaji"/>
    </item>
    <item>
     <widget class="QLineEdit" name="nik"/>
    </item>
    <item>
     <widget class="QLineEdit" name="tunjangan"/>
    </item>
    <item>
     <widget class="QLineEdit" name="total_lembur"/>
    </item>
    <item>
     <widget class="QLineEdit" name="total_gaji"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>383</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btnTambah">
      <property name="text">
       <string>Tambah</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEdit">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBersih">
      <property name="text">
       <string>BERSIH</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHapus">
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>160</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kode Gaji</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nik</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tunjangan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total Lembur</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total Gaji</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>301</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GAJI KARYAWAN</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableGaji">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>381</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>kd_gaji</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>nik</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>tunjangan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>total_lembur</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>total_gaji</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="editcari">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>239</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>158</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cari Gaji</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
